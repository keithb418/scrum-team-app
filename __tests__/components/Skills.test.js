import React from 'react';
import { shallow } from 'enzyme';
import teamMemberData from '../fixtures/teamMembers';
import Skills from '../../front-end/src/components/Skills';

test('should render the Skills component correctly', () => {
    const wrapper = shallow(<Skills member={teamMemberData[1]} />);
    expect(wrapper).toMatchSnapshot();
});

