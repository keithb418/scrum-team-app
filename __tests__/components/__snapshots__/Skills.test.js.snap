// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Skills component correctly 1`] = `
<Col
  bsClass="col"
  componentClass="div"
  md={6}
>
  <h3>
    Skills
  </h3>
  <ul
    className="member-info inline"
  >
    <li
      key="0"
    >
      react
    </li>
  </ul>
</Col>
`;
